### DISCUSSION TITLE
Having trouble splitting ExternLinksFilePaths

### DISCUSSION THREAD

**Discussion by cumiller** [Doraku/DefaultDocumentation#72] [General]
I think the command line version is not splitting the extern links argument on the "|" delimiter. The arguments parser seems to put the whole extern links argument into the first element of the string array.

**Comment by Doraku** [comment]
ah yes the format for the csproj properties (msbuild task) and the dotnet tool is different. You can directly put them like this: `-e "firstpath" "secondpath"` if you use the dotnet tool.; I will update the documentation.

**Reply by cumiller** [reply]
I wrote up an issue. The command line parser supports a delimiter/separator. #73 

Adding the separator  to the Option attribute will allow the same format to work in both cases.
